src/base/Context.ts

```diff
@@ -523,6 +523,30 @@ export class Context {
     return invalidDids;
   }
 
+  /**
+   * Returns an Identity when given a did string
+   *
+   * @throws if the Identity does not exist
+   */
+  public async getIdentity(identity: Identity | string): Promise<Identity> {
```

Comment:
This should also be used in the `getIdentity` method in the root class

Reply:
Update that to call this method